options.go

```diff
@@ -61,6 +61,14 @@ func NoDefaultHelp() Option {
 	})
 }
 
+// PostBuild allows you to add post-build options to the Kong context.
```

Comment:
This would be better described as:

> PostBuild provides read/write access to kong.Kong after initial building is complete but before parsing occurs.
>
> This is useful for eg.  adding short options to flags, updating help, etc.

Reply:
Updated the function description as suggested.